### PR TITLE
Add zfit lineshape proposal

### PR DESCRIPTION
**Pull request by mayou36**
Hi all, this is a proposal for a project on zfit.

(I forgot to PR the proposal yesterday.)

### CODE REVIEW

File: _gsocproposals/2021/proposal_zfit_lineshape [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+ * The function is finally wrapped inside a PDF and, together with tests and docs, added to the zfit library.
+
+## Expected results
+Working faddeeva function, implemented using the low-level functionality of TensorFLow (which is Numpy-like).
+
+## Requirements
+Python, Numpy, (evtl C++ and CUDA)
+
+## Mentors
+  * [Jonas Eschle](mailto:Jonas.Eschle@cern.ch)
```
[the remark this answers is not in the document]
Comment (agheata) [reply _gsocproposals/2021/proposal_zfit_lineshape:35]: The main mentor has to be in **bold**. You have to add a backup mentor (or more, but at least one). All mentors have to appear in gsoc/2021/mentors.md

File: _gsocproposals/2021/proposal_zfit_lineshape [hunk 2]
```diff
@@ -0,0 +1,41 @@
+---
+title: Implementation of physical shape function
+layout: gsoc_proposal
+project: ROOT
+year: 2018
+organization: University of Zurich
```
[the remark this answers is not in the document]
Comment (agheata) [reply _gsocproposals/2021/proposal_zfit_lineshape:6]: There are examples in that folder. Don't forget to add the logo file for the organization.

**Review by agheata — CHANGES_REQUESTED** [review]

File: _gsocproposals/2021/proposal_zfit_lineshape (hunk 2 above)
[the remark this answers is not in the document]
Comment (mayou36) [reply _gsocproposals/2021/proposal_zfit_lineshape]: I've re-used an older org file and called the org "UZH", I assume it's fine?

File: _gsocproposals/2021/proposal_zfit_lineshape (hunk 1 above)
[the remark this answers is not in the document]
Comment (mayou36) [reply _gsocproposals/2021/proposal_zfit_lineshape]: Added Rafael

[the remark this answers is not in the document]
Comment (mayou36) [reply _gsocproposals/2021/proposal_zfit_lineshape]: Also added to the mentors

File: _gsocproposals/2021/proposal_zfit_lineshape (hunk 2 above)
[the remark this answers is not in the document]
Comment (agheata) [reply _gsocproposals/2021/proposal_zfit_lineshape]: Yes

File: _gsocproposals/2021/proposal_zfit_lineshape.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+ * The function is finally wrapped inside a PDF and, together with tests and docs, added to the zfit library.
+
+## Expected results
+Working faddeeva function, implemented using the low-level functionality of TensorFLow (which is Numpy-like).
+
+## Requirements
+Python, Numpy, (maybe C++ and CUDA)
+
+## Mentors
+  * **[Jonas Eschle](mailto:Jonas.Eschle@cern.ch)**
```
Comment (agheata) [inline _gsocproposals/2021/proposal_zfit_lineshape.md:36]: Only the main mentor in bold, please. We need only one responsible, for the evaluations, but we'll keep all mentors in the loop.

File: gsoc/2021/mentors.md [hunk 4]
```diff
@@ -65,3 +65,5 @@ layout: plain
 * Pim Verschuuren [pim.verschuuren@rhul.ac.uk](mailto:pim.verschuuren@rhul.ac.uk) Royal Holloway University of London
 * Valentin Volkl [valentin.volkl@cern.ch](mailto:valentin.volkl@cern.ch) CERN
 * Omar Zapata [Omar.Zapata@cern..ch](mailto:Omar.Zapata@cern.ch) CERN
+* Jonas Eschle [Jonas Eschle](mailto:Jonas.Eschle@cern.ch) UZH
+* Rafael Silva Coutinho [Rafael Silva Coutinho](mailto:rafael.silva.coutinho@cern.ch) UZH
```
Comment (agheata) [inline gsoc/2021/mentors.md:69]: Insert sorted by last name in the list

**Review by agheata — CHANGES_REQUESTED** [review]

**Comment by mayou36** [comment]
Okay done everything. Sorry for the trivial mistakes, I fixed them before but committed to the wrong branch...

**Review by agheata — APPROVED** [review]

**Pull request merged**